cylc/flow/scripts/workflow_state.py

```diff
@@ -64,8 +65,6 @@
 
 Warnings:
   - Typos in the workflow or task ID will result in fruitless polling.
-  - To avoid missing transient states ("submitted", "running") poll for the
-    corresponding output trigger instead ("submitted", "started").
```

Comment:
This doesn't belong under warnings anymore as we now refuse to poll for transient statuses. I've added the advice on final statuses and triggers to the main text above.